### PR TITLE
✨ Introduce user permissions

### PR DESCRIPTION
**Pull request by malted**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -217,6 +218,10 @@ def onboarding?
     full_name.blank?
   end
 
+  def position(event)
```
Comment (northeastprince) [inline app/models/user.rb]:
```suggestion
  def position_in(event)
```

File: app/models/user.rb [hunk 2]
```diff
@@ -170,8 +170,9 @@ def initials
     words.any? ? words.map(&:first).join.upcase : name
   end
 
-  def pretty_phone_number
-    Phonelib.parse(self.phone_number).national
+  def pretty_phone_number(international: false)
+    parsed_number = Phonelib.parse(self.phone_number)
+    international ? parsed_number.international : parsed_number.national
   end
```
Comment (northeastprince) [inline app/models/user.rb]: Unrelated
```suggestion
  def pretty_phone_number(international: false)
    Phonelib.parse(self.phone_number).national
  end
```

**Comment by malted** [comment]
Replaced by #5201
